mllib/src/test/scala/org/apache/spark/ml/optim/loss/RDDLossFunctionSuite.scala

```diff
@@ -46,11 +46,11 @@ class RDDLossFunctionSuite extends SparkFunSuite with MLlibTestSparkContext {
     val lossWithReg = new RDDLossFunction(instances, getAgg, Some(regLossFun))
 
     val (loss1, grad1) = lossNoReg.calculate(coefficients.asBreeze.toDenseVector)
-    val (regLoss, regGrad) = regLossFun.calculate(coefficients.toArray)
+    val (regLoss, regGrad) = regLossFun.calculate(coefficients)
```

Comment:
Again this serves to highlight the inconsistency - the loss fn and reg loss fn should be composable IMO.